### PR TITLE
setfile: add page

### PR DESCRIPTION
**Pull request by harleyjwilson**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** latest

### CODE REVIEW

**Comment by harleyjwilson** [comment]
I got there in the end with the white spaces. Linked to issue #7891.

File: pages/osx/setfile.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# setfile
+
+> Sets file attributes on files in an HFS+ directory.
+> More information: <https://ss64.com/osx/setfile.html>.
+
+- Set creation date for file:
+
+`setfile -d "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/setfile.md:8]:
```suggestion
- Set creation date for specific files:

`setfile -d "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file1 path/to/file2 ...}}`
```

Optional fix: `a specific` may sound better.
Optional fix: [option specification](https://www.manpagez.com/man/1/SetFile/) violation: multiple files can be passed.

File: pages/osx/setfile.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Set modification date for file:
+
+`setfile -m "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/setfile.md:12]:
```suggestion
- Set modification date for specific files:

`setfile -m "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file1 path/to/file2 ...}}`
```

The same issue as above.

File: pages/osx/setfile.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Set modification date for symlink file (not to linked file itself):
+
+`setfile -P -m "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file}}`
```
Comment (EmilySeville7cfg) [inline pages/osx/setfile.md:16]:
```suggestion
- Set modification date for symlink file (not to linked file itself):

`setfile -P -m "{{MM/DD/YYYY HH:MM:SS}}" {{path/to/file1 path/to/file2 ...}}`
```

The same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]
- No mandatory changes found, approved.
- This page should be reviewed by `setfile` users.
- Thanks for a contribution! 😄

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Comment by kbdharun** [comment]
@blueskyson I think the commit message wasn't cleared. Check it in the future thanks.

**Pull request merged**